get-started/orchestration.md

```diff
@@ -43,7 +43,7 @@ Docker Desktop will set up Kubernetes for you quickly and easily. Follow the set
     spec:
       containers:
       - name: testpod
-        image: alpine:3.5
+        image: alpine:latest
```

Comment:
Ah, I recall seeing that PR; we should update such examples indeed, but perhaps instead of `latest` we should make sure to still use an (up-to-date) version, as "use `:latest` is not best practice.

Reply:
@thaJeztah Thanks. Agree adding 'latest' may not be best practice. Thought it would be OK in the getting started context.